src/SelfAssessment/AssessmentStart.tsx

```diff
@@ -50,3 +50,4 @@ const style = StyleSheet.create({
 })
 
 export default AssessmentStart
+
```

Comment:
This might be failing the validation

Reply:
you can run `yarn format` to run prettier for every file.

Reply:
Yep!